rep-0003.rst

```diff
@@ -169,14 +200,12 @@ These targets, including starting and ending support dates, are based on the Dis
 C++
 ---
 
-We use the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
+As of January 2015 (Hydro, Indigo, and Jade), we use the C++03 (ISO/IEC 14882:2003) standard, and are compiler-agnostic.
```

Comment:
I think the best thing to do is reference the rosdistros to which this applies skipping the date. Like below https://github.com/ros-infrastructure/rep/pull/93/files#diff-41ae062315c36c45afaa5c192b2ca7c5R247